### ISSUE TITLE
Asklepios (Carmine-tailed Janberry) Drop Rate Incorrect

### ISSUE THREAD

**Issue by VirtuosusFFXI** [EdenServer/community#3260]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

### Details

version 30181205_0

According to ffxidb, the drop rate is 94% out of a sample size of 298:
http://www.ffxidb.com/zones/160/carmine-tailed-janberry

According to the allakhazam page, comments seem to indicate that this should be either a 100% drop rate or damn near close to it, similar to Jelly Ring:
https://ffxi.allakhazam.com/db/item.html?fitem=3781

On Eden, using a combination of a friend's kills as well as some kills I did on the test server, currently have observed 1 Asklepios out of 15 kills, with various levels of TH. Even though the sample size isn't huge, its plain to see that the drop rate is nowhere near the reported 94% on ffxidb, nor does it match up with the comments on alla.

My suggestion would be to make the drop rate the same as Jelly Ring, where it is nearly guaranteed to drop every time.

**Comment by setsuk0** [comment]
I know personal experiences tend to not mean a whole lot around here when we want actual data, but I will say that I killed this NM a number of times on retail for three of my own characters as well as for a number of my friends because for whatever reason we always wanted rare/ex stuff/thought it was cool. I have at least 10 kills on retail (2004-2006) without treasure hunter (but likely way more than 10) and always got the drop/never saw asklepios not drop. I tried searching various forums as well and everything I found that mentioned drop rate also seemed to imply it should be near 100% drop rate.
